src/HubSpotException.cs

```diff
@@ -1,12 +1,15 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Net.Http;
 using System.Text;
 
 namespace Skarp.HubSpotClient
 {
     [Serializable]
     public class HubSpotException : Exception
     {
+        private HttpResponseMessage response;
```

Comment:
it would probably be better to have this as a public read-only property to make it accessible for consumers ?